app/main.py

```diff
@@ -1,3 +1,55 @@
+from __future__ import annotations
+import math
+
+
 class Vector:
-    # write your code here
-    pass
+    def __init__(self, xx: int, yy: int) -> None:
```

Comment:
Będzie bardziej czytelnie :>

```suggestion
    def __init__(self, x_axis: int, y_axis: int) -> None:
```